chain/client/src/client.rs

```diff
@@ -669,16 +691,18 @@ impl Client {
 
         debug!(
             target: "client",
-            "Produced chunk at height {} for shard {} with {} txs and {} receipts, I'm {}, chunk_hash: {}",
+            "Produced chunk at height {} for shard {} with {} txs and {} receipts, I'm {}, chunk_hash: {} prev_block_hash: {}",
             next_height,
             shard_id,
             num_filtered_transactions,
             outgoing_receipts.len(),
             validator_signer.validator_id(),
             encoded_chunk.chunk_hash().0,
+            prev_block_hash,
```

Comment:
If we are touching this log line anyway, than maybe change format string to instead record structured kv pair? 

https://github.com/near/nearcore/blob/master/docs/style.md#tracing

Reply:
done.